src/index.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import Survey from './app/list/Survey';
 import ExamGeneratorView from './components/survey/Survey';
```

Comment:
When you really commit please make sure that you don't have useless stuff inside index.ts